.github/workflows/build-pr.yml

```diff
@@ -88,3 +89,45 @@ jobs:
           name: e2e-logs
           path: |
             logs-*.json        
+  
+  build_test_aks_e2e:
+    name: "Build and run e2e Test on AKS"
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      matrix:
+        KUBERNETES_VERSION: ["1.24.6"]
+        GATEKEEPER_VERSION: ["3.10.0", "3.11.0"]
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
```

Comment:
The rest of this workflow uses tags instead of digest. To maintain consistency, I would recommend either digest/tags across all.

Reply:
Good call, I've updated to tag now